### ISSUE TITLE
GlassBR - Correct Comments in Python

### ISSUE THREAD

**Issue by samm82** [smiths/caseStudies#86]
In Python, comments are denoted by `#comment`, not by `"""comment"""`. Yes triple quotes can act as comments, but they actually create a multi-line string that is garbage collected if not used (ie. assigned to a variable, etc.). Depending on their placement, they can actually mess up the code. Comments throughout the Python Implementation in GlassBR (and possible other examples as well) should be replaced to reflect this, simply to follow convention if not to avoid errors.

**Comment by smiths** [comment]
Great point @samm82.  I wasn't aware of this, but it explains some problems I've had with Python when trying to comment out blocks of code for debugging purposes.  (I was using the triple quote.)

I checked with PEP8, and they ask for comment blocks to be done with `#` at the start of each line.

Can you please create an issue for everyone that has Python code in their example asking them to replace the block comments with `#` comments.  The exception to this is documentation strings for all public modules, functions, classes, and methods.

We should also eventually update the comments in the Python code so that doxygen is supported.  I think we'll hold off on doing that for now though, since it isn't a priority.  I also wonder if the design of some of the case studies might still need some modification.

**Comment by samm82** [comment]
@smiths When I make these issues, do you want to be assigned or should I assign the appropriate team members?

**Comment by smiths** [comment]
Yes, please look at the swhs repo as well.  In all cases, assign the issue to the person currently working on the case study.  You can @ me and refer back to the original issue, so that everyone knows that I agree with the creation of these issues.

Doesn't glassbr use Python?  Are you saying that the Python code in glassbr doesn't use the triple comments?

**Comment by samm82** [comment]
No it does - that's what this issue is

**Comment by samm82** [comment]
@smiths This issue can be closed now - should I make an issue about doxygen just so that we don't lose it?
